xc7/primitives/slicem/srl/srl32.pb_type.xml

```diff
@@ -0,0 +1,32 @@
+<pb_type name="SRL32" num_pb="1" blif_model=".subckt SRL32" xmlns:xi="http://www.w3.org/2001/XInclude">
+  <clock name="CLK" num_pins="1"/>
+  <input name="CE" num_pins="1"/>  
+  <input name="A6" num_pins="1"/>  
```

Comment:
SRLC32E has not A6 pin?   Tech library shows the SRL32 as having the following inputs:

 - Q
 - Q31
 - D
 - CLK
 - CE
 - A[4:0]

Reply:
This is fixed now.